unittesting/test_format.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+from synth_data_module import FormatOutput, get_procedure_list, get_diagnosis_list
+
+
+class FormatTest(unittest.TestCase):
+
+    output_loc = 'output/'
+    encounter_type = 'inpatient'
+    fmt = FormatOutput('010735', output_loc, encounter_type)
+
+    def test_add_procedure(self):
+        pass
+        # For this test try testing without modifying the original function in format
+
+    def test_add_diagnosis(self):
+        pass
+        # For this test, first try separating the function into separate functions based on functionality.
+        # Ex: pulling data from csv vs. merging data
+
+    # def test_modify_row(self):
+    #     testdict = [{'a': 1, 'b': 2, 'c': 3},
+    #               {'a': 100, 'b': 200, 'c': 300},
+    #               {'a': 1000, 'b': 2000, 'c': 3000}]
+
+    def test_get_procedure_list(self):
+        procedure_list = get_procedure_list()
+        self.assertEqual(len(procedure_list), 48)
+
+        for i in range(48):
```

Comment:
This test is more complicated than the code it's testing! Ideally our tests are so simple that they're obviously correct, so that we don't need to write tests for the tests.

I'd recommend building the table to be human readable:
[{'Principal Procedure Code', 'Principal Procedure Date',
'Procedure Code 1' 'Procedure Date 1' 
'Procedure Code 2', 'Procedure Date 2']